src/main/resources/assets/tfc/models/block/bellows.json

```diff
@@ -0,0 +1,78 @@
+{
+  "credit": "Made with Blockbench",
```

Comment:
please import this into blockbench and save it, their newer settings have better minification than this